lab-padget/service/auth-service.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+let __API_URL__;
```

Comment:
This would generally be bad practice. You can either use an eslint ignore statement (check the docs), and you can add it as a global var to the .eslintrc file (also in the docs).